app/views/local_authorities/index.html.erb

```diff
@@ -10,7 +10,9 @@
   <table class="table table-striped table-hover table-bordered contacts-table" data-module="filterable-table">
     <thead>
       <tr class="table-header">
-        <th>Local Authorities (<%= @authorities.count %>)</th>
+        <th>
+          <div class='count'><span><%= @authorities.count %></span> local authorities</div>
```

Comment:
the `span` element here gives us the ability to target the numeric bit of the sentence for fancy CSS formatting.